mlrun/runtimes/databricks_job/databricks_wrapper.py

```diff
@@ -80,21 +102,28 @@ def run_mlrun_databricks_job(
     timeout_minutes = task_parameters.get("timeout_minutes", 20)
     number_of_workers = task_parameters.get("number_of_workers", 1)
     new_cluster_spec = task_parameters.get("new_cluster_spec")
+    artifact_json_path = task_parameters.get("artifact_json_path")
     databricks_run_name = task_parameters.get(
         "databricks_run_name", f"mlrun_task_{mlrun_databricks_job_id}"
     )
-
+    current_time = datetime.datetime.utcnow()
+    run_time = current_time.strftime("%H_%M_%S_%f")
+    databricks_run_name = (
+        databricks_run_name
+        if str(mlrun_databricks_job_id) in databricks_run_name
+        else f"{databricks_run_name}_{run_time}"
+    )
```

Comment:
Why not always use the `mlrun_databricks_job_id`, like you do if the user didn't provide a run name? Why in this case use the time?